go.mod

```diff
@@ -9,11 +9,11 @@ require (
 	github.com/Masterminds/semver/v3 v3.2.1
 	github.com/athanorlabs/go-dleq v0.1.0
 	github.com/athanorlabs/go-p2p-net v0.2.0
-	github.com/athanorlabs/go-relayer v0.1.0
+	github.com/athanorlabs/go-relayer v0.1.1-0.20230426015202-a005fed78fda
```

Comment:
Will be updated to tagged release after https://github.com/AthanorLabs/go-relayer/pull/13 is merged.